ml-agents/mlagents/trainers/tests/test_bcmodule.py

```diff
@@ -77,7 +77,7 @@ def create_ppo_policy_with_bc_mock(
     trainer_parameters["pretraining"]["demo_path"] = (
         os.path.dirname(os.path.abspath(__file__)) + "/" + demo_file
     )
-    policy = PPOPolicy(0, mock_brain, trainer_parameters, False, False)
+    policy = PPOPolicy(0, mock_brain, trainer_parameters, False, False, False)
```

Comment:
Did you run the Pytest ? You added a False here but I don't know why, it seems to break the tests.

Reply:
Was still in progress of a major change. Fixed now.